Objective-C/CBLConflict.h

```diff
@@ -27,6 +27,9 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @interface CBLConflict : NSObject
 
+/** The conflict resolved document id. */
```

Comment:
I think either just "document id" or "The document id of the conflicted documents".